public/favicons/polkadot/browserconfig.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<browserconfig>
+    <msapplication>
+        <tile>
+            <square150x150logo src="/mstile-150x150.png"/>
+            <TileColor>#da532c</TileColor>
```

Comment:
Why is this orange?

Reply:
Just realized this is the suggested color for the Windows Metro UI and I didn't edit it,  now I know that 3 favicons are using the same tilecolor.
And for the kusama and westend icon, I chose the inline ones from the img folder, so let me know if i should use other's images for the favicons.